src/transformers/models/t5/modeling_tf_t5.py

```diff
@@ -19,6 +19,7 @@
 import itertools
 import math
 import warnings
+from contextlib import nullcontext
```

Comment:
This was only added in Py3.7, which means we'll now be fully incompatible with Py <= 3.6. I'm totally fine with that, but just pointing it out!

Reply:
Or you could use our `ContextManagers` and not reinvent the wheel :-p

Reply:
@Rocketknight1 nullcontexts removed, thanks to Sylvain's suggestion :D